### PR TITLE
reference.conf: Add an example for configuring the ScanCode command line

### PR DESCRIPTION
**Pull request by sschuberth**
Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

**Review by mnonnenmacher — APPROVED** [review]

**Comment by sschuberth** [comment]
Note that this PR actually depends on the option naming in https://github.com/oss-review-toolkit/ort/pull/3564/files#diff-c767f2e80745238bc7d39cd2f6217e621bfcf3ec9db9e1c8bacafdd36eb9570eR203, so I'll wait until that other PR is merged first.

**Pull request merged**
